x11-libs/libice/libice-1.0.9.recipe

```diff
@@ -0,0 +1,74 @@
+SUMMARY="Inter-Client Exchange Library"
+DESCRIPTION="The ICE library utilizes callbacks to process incoming \
+messages. Using callbacks allows ProtocolSetup messages and authentication \
+to happen behind the scenes. An additional benefit is that messages never \
+need to be buffered up by the library when the client blocks waiting for \
+a particular message."
+HOMEPAGE="http://xorg.freedesktop.org/releases/individual/lib/"
+COPYRIGHT="1993, 1998 The Open Group"
+LICENSE="MIT (no promotion)"
+REVISION="1"
+SOURCE_URI="https://www.x.org/releases/individual/lib/libICE-$portVersion.tar.gz"
+CHECKSUM_SHA256="7812a824a66dd654c830d21982749b3b563d9c2dfe0b88b203cefc14a891edc0"
+SOURCE_FILENAME="libICE-$portVersion.tar.gz"
+SOURCE_DIR="libICE-$portVersion"
+
+ARCHITECTURES="!x86_gcc2 x86 ?x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+libVersion=6.3.0
+libVersionCompat="$libVersion compat >= 6"
+
+PROVIDES="
+	libice$secondaryArchSuffix = $portVersion
+	lib:libICE$secondaryArchSuffix = $libVersionCompat
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	libice${secondaryArchSuffix}_devel = $portVersion
+	devel:libICE$secondaryArchSuffix = $libVersion
+	"
+REQUIRES_devel="
+	libice$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:xproto$secondaryArchSuffix
+	devel:xtrans$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	devel:util_macros$secondaryArchSuffix
+	cmd:aclocal
+	cmd:autoconf
+	cmd:libtoolize$secondaryArchSuffix
+	cmd:make
+	cmd:gcc$secondaryArchSuffix
+	cmd:pkg_config$secondaryArchSuffix
+	"
+
+defineDebugInfoPackage libice$secondaryArchSuffix \
+	$libDir/libICE.so.$libVersion
+
+BUILD()
+{
+	autoreconf -vfi
+	runConfigure ./configure
```

Comment:
Did you forgot to run anything?

Reply:
As? libtoolize is called by autoreconf if you mean that?

Reply:
make?

Reply:
Jikes! (as I told, it was a base from one of the other recipes) didn't notice that one! :+1:

Reply:
Maybe you should check the original recipe and fix that too ;)

Reply:
Not sure (didn't check) who wrote the initial recipes :) but will check :)